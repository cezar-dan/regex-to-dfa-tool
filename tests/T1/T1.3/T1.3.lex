c
C
0
0,'c',1
1,'c',2
1

ab
ABS
0
0,'a',1
0,'b',2
1,'a',2
1,'b',3
3,'a',1
3,'b',2
3

b
BS
0
0,'b',1
1,'b',1
1 0