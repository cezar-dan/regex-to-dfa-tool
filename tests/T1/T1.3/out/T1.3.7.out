ABS abab
C c
BS bbb
C c
BS b
ABS ab
BS b
C c
ABS ab
BS b
ABS ab